contracts/layer-2/PaymentRelay.sol

```diff
@@ -9,18 +9,36 @@ import "solidity-bytes-utils/contracts/BytesLib.sol";
 
 /**
  * @title PaymentRelay
- * @dev Allow to sent payment with offchain triggers only once.
+ * @dev Allows to sent payment with offchain triggers through a two-stage payment flow. This is intended
+ * to allow flow for payment refunds.
+ *
+ * The payment flow is as follows:
+ * 1 - Funds owner calls `reservePayment`, placing his funds in escrow and creating a new payment reservation;
+ * 2 - Funds owner, or authorized operator, calls `execPayment` to forward the funds to an approved recipient;
+ * 3 - In case the payment's offchain transaction fails, an authorized operator calls `refundPayment` to trigger
+ * an escrow refund to the payment's originator
+ *
+ * For calling `execPayment` or `refundPayment`, the caller must have EXECUTION_ROLE, or REFUND_ROLE, respectively.
```

Comment:
P1
Wording about EXECUTION_ROLE & REFUND_ROLE in the smartcontract, but they have been replaced by OPERATOR_ROLE